### PR TITLE
feat: be more descriptive about what is done in `issue update` and `mr update`

### LINKED ISSUE #461: be more descriptive of what is done when updating an issue/mr
**Describe the feature or problem you'd like to solve**

When updating an issue or merge request with `issue update` or `mr update` it would be nicer if we told the user what exactly was done.

**Propose a Solution**

Instead of a blank `Updated` a `changed title to FOO` or `made confidential`. for long things like descriptions I think we can get away with `updated description` but for short things like locking/unlocking a discussion we can affort to say `locked discussion on $ISSUE`

**Additional context**

This is done in gh and help re-assure the user that it didn't mess up anything, nothing worse than doing X and then finding out actually it was Y that happened!

### PR DESCRIPTION
**Pull request by maxice8**
**Description**

See screenshots

**Related Issue**

Resolves #461

**How Has This Been Tested?**

See screenshots

**Screenshots (if appropriate):**
issue-update-descriptive-1
issue-update-descriptive-2

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation
- [ ] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

File: commands/issue/update/issue_update_test.go [hunk 1]
```diff
@@ -60,9 +60,16 @@ func TestNewCmdUpdate(t *testing.T) {
...
-			ExpectedMsg: []string{"- Updating issue #1", "✓ Updated", "#1 New Title"},
+			Name:  "Issue Exists",
+			Issue: `1 -t "New Title" -d "A new description" --lock-discussion -l newLabel --unlabel bug`,
+			ExpectedMsg: []string{
+				"- Updating issue #1",
+				"updated title to \"A new description\"",
+				"locked discussion",
+				"added labels newLabel",
+				"removed labels bug",
+				"✓ Updated", "#1 New Title",
```
Comment (profclems) [inline commands/issue/update/issue_update_test.go]:
```suggestion
				"#1 New Title",
```

File: commands/issue/update/issue_update_test.go [hunk 2]
```diff
@@ -60,9 +60,16 @@ func TestNewCmdUpdate(t *testing.T) {
...
 	}{
 		{
-			Name:        "Issue Exists",
-			Issue:       `1 -t "New Title" -d "A new description" --lock-discussion -l newLabel --unlabel bug`,
-			ExpectedMsg: []string{"- Updating issue #1", "✓ Updated", "#1 New Title"},
+			Name:  "Issue Exists",
+			Issue: `1 -t "New Title" -d "A new description" --lock-discussion -l newLabel --unlabel bug`,
+			ExpectedMsg: []string{
+				"- Updating issue #1",
+				"updated title to \"A new description\"",
```
Comment (profclems) [inline commands/issue/update/issue_update_test.go]:
```suggestion
				"✓ updated title to \"New Title\"",
```

**Comment by maxice8** [comment]
The tests are still failing and the error message isn't that helpful

**Comment by maxice8** [comment]
derp, typo

**Review by profclems — APPROVED** [review]
🎉

**Pull request merged**